trappy/plotter/js/ILinePlot.js

```diff
@@ -123,6 +123,7 @@ var ILinePlot = ( function() {
             stepPlot: t_info.step_plot,
             logscale: t_info.logscale,
             fillGraph: t_info.fill_graph,
+            fillAlpha: t_info.fill_alpha,
```

Comment:
looks like this only works properly in Chrome.  In Firefox the alpha seems to be the same if I use 0.1 or 0.9.  Anyway, that is a dygraph bug, not trappy.